test/clj/rems/api/services/test_resources.clj

```diff
@@ -36,28 +38,28 @@
                                         :enabled true})
       (is (= {:enabled true
               :archived false}
-             (status-flags res-id))))
+             (status-flags res-id owner))))
```

Comment:
I don't like having to pass the current user everywhere. It seems rather wasteful compared to a long time ago when we used the dynamic scope / context.

Reply:
`status-flags` is only a util for this test. I guess one option would be to move `status-flags` inside the `let` so it can use `owner` directly :)

Reply:
Done.